src/assets/styles/main.scss

```diff
@@ -38,7 +38,12 @@ $icon-font-path: '../icon-font' !default;
     &:focus,
     &:focus:before {
       outline: none;
-      text-shadow: 0 0 1em var(--color-text);
+      text-shadow:
+        0 0 0.1em var(--color-accent),
+        0 0 0.2em var(--color-accent),
+        0 0 0.3em var(--color-accent),
+        0 0 0.8em var(--color-text),
+        0 0 1em var(--color-text);
```

Comment:
Added some accent to our focus highlight following [this feedback](https://cloudinary.slack.com/archives/C9B1DMFV3/p1741613105574709).
Didn't mean to push it to this PR but I'll let it slide...